.github/workflows/pr_test.yml

```
@@ -1,4 +1,9 @@
-on: pull_request
+on: 
+  pull_request:
+    types:
+      - opened
+      - synchronize
+      - reopened
```

```suggestion
  pull_request:
    types:
      - opened
      - synchronize
      - reopened
    branches:
      - main
```

Since we don't *require* the fork-and-PR workflow here, some Fastlyans may open branches directly in this repository but the workflow should not run until they open a PR.